modules/ezcontent_block/config/install/field.storage.block_content.field_media.yml

```diff
@@ -0,0 +1,19 @@
+langcode: en
+status: true
+dependencies:
+  module:
+    - block_content
+    - media
+id: block_content.field_media
+field_name: field_media
+entity_type: block_content
+type: entity_reference
+settings:
+  target_type: media
+module: core
+locked: false
+cardinality: 1
+translatable: true
+indexes: {  }
+persist_with_no_fields: false
+custom_storage: false
```

Comment:
If we plan to reuse this field in any other component then its good to be here and we would need to send out a configuration update for this. If not, then let's add this file inside install directory of newly created module -- ezcontent_block_media